extensions/NumberFormat/NumberFormat.php

```diff
@@ -27,7 +27,7 @@ function number_format_Setup( Parser $parser )	 {
 
 class NumberFormart {
 
-	public static function number_format_Render( &$parser ) {
+	public static function number_format_Render( $parser ) {
 		wfProfileIn(__METHOD__);
 		// {{#number_format:number|decimals|dec_point|thousands_sep}}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)